src/radical/pilot/configs/resource_ornl.json

```diff
@@ -181,6 +181,56 @@
                                          "options": ["gpumps", "nvme"]}
     },
 
+    "summit_interactive": {
+        "description"                 : "4608 nodes with 2 IBM POWER9 CPUs and 6 NVIDIA Volta V100 GPUs",
+        "notes"                       : "interactive job https://docs.olcf.ornl.gov/systems/summit_user_guide.html",
+        "schemas"                     : ["interactive"],
+        "interactive"                 : {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+        "resource_manager"            : "LSF",
+        "agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["JSRUN"],
+                                         "JSRUN": {
+                                             "pre_exec_cached": [
+                                               # "module unload xl",
+                                               # "module unload xalt"
+                                             ]
+                                         }
+                                        },
+        "pre_bootstrap_0"             : [
+                                         "module unload py-pip",
+                                         "module unload py-virtualenv",
+                                         "module unload py-setuptools",
+                                         "module unload xl",
+                                         "module unload xalt",
+                                         "module load   gcc/9.1.0",
+                                         "module load   libzmq/4.3.3",
+                                         "module load   python/3.7-anaconda3"
+                                        ],
+        "pre_bootstrap_1"             : ["module load   gcc/9.1.0",
+                                         "module load   libzmq/4.3.3",
+                                         "module load   python/3.7-anaconda3",
+                                         "ulimit -u 65536"
+                                        ],
+        "valid_roots"                 : ["$MEMBERWORK/"],
+        "default_remote_workdir"      : "$MEMBERWORK/${LSB_PROJECT_NAME,,}",
+        "python_dist"                 : "default",
+        "virtenv_dist"                : "default",
+        "virtenv_mode"                : "local",
+        "rp_version"                  : "local",
```

Comment:
just keep only one parameter: `"virtenv_mode"                : "local"` (other parameters will be set by RP)